compose/snippets/src/main/java/com/example/compose/snippets/text/TextSnippets.kt

```diff
@@ -591,188 +581,51 @@ private object TextEffectiveStateManagement2 {
     // [END android_compose_text_state_management]
 }
 
+// [START android_compose_text_link_1]
 @Composable
-private fun TextSample(samples: Map<String, @Composable ()->Unit>) {
-    MaterialTheme {
-        Box(
-            Modifier
-                .background(MaterialTheme.colorScheme.background)
-                .fillMaxWidth()
-        ) {
-            Column(
-                verticalArrangement = Arrangement.spacedBy(10.dp),
-                modifier = Modifier.padding(10.dp)
+fun AnnotatedStringWithLinkSample() {
+    // Display a link in the text
+    Text(
+        buildAnnotatedString {
+            append("Build better apps faster with ")
+            withLink(
+                LinkAnnotation.Url(
+                    "https://developer.android.com/jetpack/compose",
+                    TextLinkStyles(style = SpanStyle(color = Color.Blue))
+                )
             ) {
-                samples.forEach { (title, content) ->
-                    Row(Modifier.fillMaxWidth()) {
-                        content()
-                        Text(
-                            text = title,
-                            textAlign = TextAlign.Center,
-                            style = MaterialTheme.typography.titleLarge,
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .align(Alignment.CenterVertically)
-                        )
-                    }
-                }
+                append("Jetpack Compose")
             }
         }
-    }
-}
-
-private const val SAMPLE_LONG_TEXT =
-    "Jetpack Compose is Android’s modern toolkit for building native UI. " +
-        "It simplifies and accelerates UI development on Android bringing your apps " +
-        "to life with less code, powerful tools, and intuitive Kotlin APIs. " +
-        "It makes building Android UI faster and easier."
-@Composable
-@Preview
-fun LineBreakSample() {
-    // [START android_compose_text_line_break]
```

Comment:
@bentrengrove is this correct that the latest changes are removing these snippets? The snippet bot is complaining that they are still used.

Reply:
No, it's wrong. It looks like TextSnippets changed in both latest and main and it wasn't merged correctly

Reply:
I think this will fix it https://github.com/android/snippets/pull/326